manifests/theme.pp

```diff
@@ -1,20 +1,51 @@
 define wp::theme (
+	$slug = $title,
 	$location,
 	$ensure = enabled
 ) {
-	#$name = $title,
 	include wp::cli
 
 	case $ensure {
 		enabled: {
-			$command = "activate $title"
+			exec { "wp install theme $title --activate$network$held":
+				cwd     => $location,
+				user    => $::wp::user,
+				command => "/usr/bin/wp theme install $slug --activate $held",
+				unless  => "/usr/bin/wp theme is-installed $slug",
```

Comment:
This should also include a check for the current theme, otherwise it won't activate if it's already installed.